lib/lib.dom.d.ts

```diff
@@ -12916,28 +12452,6 @@ declare var SVGElement: {
     new(): SVGElement;
 };
 
-interface SVGElementInstance extends EventTarget {
-    readonly correspondingElement: SVGElement;
-    readonly correspondingUseElement: SVGUseElement;
-}
```

Comment:
These properties used to be available on `SVGElement`, which used to extend `SVGElementInstance` in [LOC 12903](https://github.com/microsoft/TypeScript/pull/44842/files#diff-796971dc74a3f3cd77ca0a3a776d6fe27ab224cd501d9652dbeffcf45ba0f058L12903). They disappeared during the upgrade from [`v4.3.5` to `v4.4.2`](https://github.com/microsoft/TypeScript/compare/v4.3.5...v4.4.2).

Reply:
Removal was intentionalm, as per https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1023#issuecomment-908524091:

> Those are not available on any modern browser nor MDN, and thus are probably IE-specific properties. The removal is intended since the types library intends to only include things that exist in modern browsers.